### PR TITLE
Metabase48

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

**Review by stepansergeevitch — APPROVED** [review]
LGTM

File: src/metabase/driver/firebolt.clj [hunk 1]
```diff
@@ -100,63 +95,44 @@
...
-
 ; Modify start of week to monday
-(defmethod sql.qp/date [:firebolt :week]
-  [_ _ expr] (sql.qp/adjust-start-of-week :firebolt (partial date-trunc :week) expr))
+(defn- to-start-of-week [expr] [:'toMonday expr])
+(defmethod sql.qp/date [:firebolt :week] [driver _ expr] (sql.qp/adjust-start-of-week driver to-start-of-week expr))
 
 ; Return a appropriate HoneySQL form for converting a Unix timestamp integer field or value to an proper SQL Timestamp.
-(defmethod sql.qp/unix-timestamp->honeysql [:firebolt :seconds] [_ _ expr] (hsql/call :to_timestamp expr))
+; (defmethod sql.qp/unix-timestamp->honeysql [:firebolt :seconds] [_ _ expr] (hsql/call :to_timestamp expr))
```
Comment (ptiurin) [inline src/metabase/driver/firebolt.clj]: comment

File: src/metabase/driver/firebolt.clj [hunk 2]
```diff
@@ -345,10 +325,6 @@
            ;; find PKs and mark them
            (add-table-pks (.getMetaData conn))))))
 
-; making between clause in a query to be inside brackets
-(defmethod hformat/fn-handler "between" [_ field lower upper]
-  (str "(" (hformat/to-sql-value field) " BETWEEN " (hformat/to-sql-value lower) " AND " (hformat/to-sql-value upper) ")"))
-
```
Comment (ptiurin) [inline src/metabase/driver/firebolt.clj:351]: Is this supposed to be removed? Would date ranges still work in this case?

File: README.md [hunk 3]
```diff
@@ -62,3 +62,14 @@ You should see a message on startup similar to:
...
+## Compatibility matrix
+
+| Firebolt Version | Metabase Release | Driver Version |
+|------------------|------------------|----------------|
+| 1                | <=0.47.x         | <=1.0.13       |
+| 1                | <=0.47.x         | <=1.0.13       |
+| 2                | <=0.47.x         | 3.0.0          |
+| 1&2              | <=0.47.x         | 3.0.1          |
+| 1&2              | 0.48.x           | 3.0.3          |
+| 1&2              | 0.49.x           | 3.0.3          |
```
Comment (ptiurin) [inline README.md:75]: Do we need this part? I think we're not using Firebolt version for customer-facing documentation. We can probably have just metabase versions and driver versions matrix.

File: src/metabase/driver/firebolt.clj (hunk 2 above)
Comment (alexradzin) [reply src/metabase/driver/firebolt.clj:351]: yes, this is what Metabase's experts told me.

File: README.md (hunk 3 above)
Comment (alexradzin) [reply README.md:75]: I believe we need. At least drivers prior 3.0.3 will not work with Metabase after 48 and it is important to mention this in documentation. All custom drivers providers do this. For example Click house.

File: src/metabase/driver/firebolt.clj (hunk 1 above)
Comment (alexradzin) [reply src/metabase/driver/firebolt.clj]: thanks

File: README.md (hunk 3 above)
Comment (ptiurin) [reply README.md:75]: Please re-read the question. I'm asking about Firebolt version, which we don't advertise to the users. Metabase release X driver version makes sense here.

**Pull request merged**